scripts/fuzz_opt.py

```diff
@@ -734,8 +734,11 @@ def randomize_opt_flags():
         choice = random.choice(opt_choices)
         if '--flatten' in ret and '--flatten' in choice:
             print('avoiding multiple --flatten in a single command, due to exponential overhead')
-        else:
-            ret += choice
+            continue
+        ret += choice
+        # add a sprinkling of round trips
+        if random.random() < 0.5:
+            ret.append('--roundtrip')
```

Comment:
How about putting it in a random position in the middle?
```python
if random.random() < 0.5:
  position = random.randint(0, len(ret))
  ret = ret[:position] + ['--roundtrip'] + ret[position:]
```

Reply:
This is a little tough because we don't want to put it between passes that need to go together like with passes that need to be preceeded by `flatten` or soon passes that need `--generate-stack-ir`.  Also, this already does add round trips at multiple random places in the middle of the opts because this code is inside the loop.

Reply:
Ah, I see, I missed that it was inside the loop, thanks.

Still this looks a little unoptimal since 0.5 means that we place quite a lot of these - like half the number of passes. that doesn't test the normal case enough (it will be rare to see runs of 4 passes one after the other without a roundtrip).

how about making a list of lists initially. so a list item could be `['--precompute']` or `['--flatten', 'something']`. then we can put the code I suggested, and then combine the lists?